evm2wasm.cpp

```diff
@@ -54,5 +52,17 @@ string evm2wasm(string input) {
 }
 
 int main(int argc, char **argv) {
-  cout << evm2wasm("600160020200") << endl;
+	if(argc<2) {
+		cout << "usage " << argv[0] << " <evm file>" << endl;
+		return 1;
+	}
+	FILE *fd = fopen(argv[1], "r");
+	fseek(fd, 0, SEEK_END);
+	size_t offset = ftell(fd);
+	rewind(fd);
+	char *code = (char*)malloc(8192);
```

Comment:
If `offset` is the file size, then why not use it here?

Reply:
oh yea I forgot to use that :D